internal/grpc/proto/kong/admin/model/v1/key.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package kong.admin.model.v1;
+
+option go_package = "github.com/kong/koko/internal/gen/grpc/kong/admin/model/v1;v1";
+
+enum KeyFormat {
+  KEY_FORMAT_UNSPECIFIED = 0;
+  KEY_FORMAT_JWK = 1;
+  KEY_FORMAT_PEM = 2;
+  KEY_FORMAT_KID = 3;
+}
+
+message Key {
+  string id = 1;
+  int32 created_at = 2;
+  int32 updated_at = 3;
+  repeated string tags = 4;
+
+  KeySet set = 5;
+  string name = 6;
+  JwkKey jwk = 7;
+  PemKey pem = 8;
+  KeyFormat key_format = 9;
```

Comment:
Is this field used at all?

Reply:
I don't think so; but we'll have to check with the Kong implementation (which i think has just been merged in Kong/kong#9737)